docs/develop/localization.md

```diff
@@ -145,6 +145,195 @@ For Outlook add-ins, the [SourceLocation] element also aligns to the form factor
+
+```json
+{
+    "resources":{ 
+        "en-us": { 
+            "SHOWTASKPANE_default_shortcut": { 
+                "value": "CTRL+SHIFT+A", 
+            }, 
+            "SHOWTASKPANE_action_name": {
+                "value": "Show task pane for add-in",
+            }, 
+        },
+        "fr-fr": { 
+            "SHOWTASKPANE_default_shortcut": { 
+                "value": "CTRL+SHIFT+A", 
+            }, 
+            "SHOWTASKPANE_action_name": {
+                "value": "Afficher le volet de tâche pour add-in",
+              } 
+        }
+    }
+}
+```
+
+Note that there is no `default` property in the file that is a peer to the `en-us` and `fr-fr` sections. This is because the default strings, which are used when the locale of the Office host application doesn't match any of the *ll-cc* properties in the resources file, *must be defined in the extended overrides file itself*. Defining the default strings directly in the extended overrides file ensures that Office doesn't have to download the resource file when the locale of the Office application matches the default locale of the add-in (as specified in the manifest). The following is a corrected version of the preceding example of an extended overrides file that uses resource tokens.
+
+```json
+{
+    "actions": [
+        {
+            "id": "SHOWTASKPANE",
+            "type": "ExecuteFunction",
+            "name": "${resource.SHOWTASKPANE_action_name}"
+        }
+    ],
+    "shortcuts": [
+        {
+            "action": "SHOWTASKPANE",
+            "key": {
+                "default": "${resource.SHOWTASKPANE_default_shortcut}"
+            }
+        }
+    ],
+    "resources": { 
+        "default": { 
+            "SHOWTASKPANE_default_shortcut": { 
+                "value": "CTRL+SHIFT+A", 
+            }, 
+            "SHOWTASKPANE_action_name": {
+                "value": "Show task pane for add-in",
+            } 
+        }
+    }
+}
+```
+
+### Method 2: Create separate extended override files for various locales or sets of locales
+
+If you prefer, you can create separate extended override files for each locale, rather than use a resource file. Although the [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element has only one `Url` attribute, it is possible for the attribute to refer to more than one file. This is because the value of the attribute doesn't have to be a literal URL. It can be a URL template with a locale placeholder, called a token. The following is an example. Note that the format of the token is **${token.*name-of-token*}**.
```

Comment:
```suggestion
If you prefer, create separate extended override files for each locale, rather than use a resource file. Although the [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element has only one `Url` attribute, it is possible for the attribute to refer to more than one file. This is because the value of the attribute doesn't have to be a literal URL. It can be a URL template with a locale placeholder, called a token. The following is an example. Note that the format of the token is **${token.*name-of-token*}**.
```

Reply:
Again, this is the 2nd of two ways, so I want to keep the "You can".

Reply:
Nope. It's not conformant with the guidance we are clearly given in the Microsoft Voice and Style. "You can" is considered "WEAK WRITING."